Formula/postgresql@15.rb

```diff
@@ -154,5 +158,6 @@ def caveats
     assert_equal (opt_lib/"postgresql").to_s, shell_output("#{bin}/pg_config --pkglibdir").chomp
     assert_equal (opt_include/"postgresql").to_s, shell_output("#{bin}/pg_config --pkgincludedir").chomp
     assert_equal (opt_include/"postgresql/server").to_s, shell_output("#{bin}/pg_config --includedir-server").chomp
+    assert_match "-I#{Formula["gettext"].opt_include}", shell_output("#{bin}/pg_config --cppflags")
```

Comment:
I wonder if what we should really be doing here is mocking the compilation of a dummy extension to double-check that it does, in fact, compile. That way we can make sure building extensions isn't broken by any further changes to this formula's dependencies.

Reply:
Agree that would be more robust (but add some complexity).